dist/editor/index.html

```diff
@@ -23,19 +23,9 @@
     <script src="jquery.min.js"></script>
     <!--{endif}-->
 
-    <!-- Lacking browser support -->
-    <script nomodule="" src="redirect-on-no-module-support.js"></script>
-    <script type="module" src="redirect-on-lacking-support.js"></script>
-
     <!-- As yet no ES6 Module support -->
     <script src="jquery-ui/jquery-ui-1.8.17.custom.min.js"></script>
 
-    <!-- Browser polyfills -->
-    <script src="../external/dom-polyfill/dom-polyfill.js"></script>
-
-    <!-- ES6+ polyfills (Babel) -->
-    <script src="../external/core-js-bundle/minified.js"></script>
-    <script src="../external/regenerator-runtime/runtime.js"></script>
 
```

Comment:
See comment just above. (These don't suffer from problem no. 1 though, so the mitigation mentioned at the end of no. 2 could work for these polyfills if you wanted to switch them to imports.)